### ISSUE TITLE
[FR]: Easily add a new post in a blog e.g `quarto create post` 

### ISSUE THREAD

**Issue by cderv** [quarto-dev/quarto-cli#3917]
This comes from https://github.com/quarto-dev/quarto-r/issues/22

R users have such feature with `blogdown::new_post()` and a RStudio Addin build upon it. I believe it would be a nice feature to have in our `quarto create ...` suites of features. 

This would help 
* Create a post in the right place in the Quarto blog
* Add some content to the file based on a default or template maybe
* Offer to fill some fields at command line maybe like title, etc...

**Comment by jthomasmock** [comment]
A few functions/packages have been created by community:

- [newpost](https://github.com/smach/newpost)
- [quarto_create_post](https://github.com/rmflight/flighttools/blob/main/R/quarto_create_post.R)
- [new_post](https://themockup.blog/posts/2022-11-08-use-r-to-generate-a-quarto-blogpost/)

But a current limitation is that all are R focused rather than general quarto CLI tooling.

**Comment by smach** [comment]
The one thing these personal R packages are missing (including mine) is the ability to see existing blog categories. It's usually not a good idea to add freeform text to what should be structured data!

**Comment by cderv** [comment]
@dragonstyle do you think we could target adding new `quarto create` feature for 1.4 ?

**Comment by petzi53** [comment]
With [quartopost](https://github.com/petzi53/quartopost) I tried to simulate the very missed blogdown "new post" RStudio Addin by Yihui Xie using a shiny dialog window with some configurations and offering a list of already used tags as commented by @smach.

**Comment by mcanouil** [comment]
I would argue that first, #5421 and #598 need to be resolved, before adding new commands.

**Comment by cderv** [comment]
@dragonstyle let's consider this for later then. 

@mcanouil yes, this would be better to clean the current state first. 

Regarding the workaround, I believe @petzi53 work on https://github.com/petzi53/quartopost is really good ! 
It is targeting RStudio's user only though but that is still a really good solution !  Thanks @petzi53 !

**Comment by astrowonk** [comment]
Yes, please! I miss `hugo new posts/something_new/index.md` and having the basic metadata like date and draft: true set for me.

**Comment by astrowonk** [comment]
I created the very simple Python based [hugo-new-content](https://github.com/astrowonk/quarto-new-content) command line utility to replicate `hugo new content` command line functionality. One can do:

```
quarto-new-content posts/star-trek-is-cool/index.qmd 
```

and it will make the new file in the new directory,  with a very minimal set of yaml headers, ready for editing.

**Comment by coatless** [comment]
For VS Code (& maybe Positron) users who miss the addin option, check out [`{quartize}`](https://github.com/coatless-vsx/quartize) ([VS Marketplace](https://marketplace.visualstudio.com/items?itemName=coatless.quartize) and [OpenVSX](https://open-vsx.org/extension/coatless/quartize)). It provides a nice form field & the option to directly specify via palette. 

![Demo workflow triggering a `new_post()` pop-up](https://github.com/user-attachments/assets/fa8982e7-0b52-4407-8db4-2d15a0aecede)
